docs/installation.md

```diff
@@ -16,12 +16,18 @@ brew tap ggozad/formulas
 brew install ggozad/formulas/oterm
 ```
 
-Using `yay` (or any AUR helper) for Arch Linux:
+Using `yay` (or any AUR helper) for Arch Linux, thanks goes to [Daniel Chesters](https://github.com/DanielChesters) for maintaining the package:
 
 ```bash
 yay -S oterm
 ```
 
+Using `nix-env` on NixOs, thanks goes to [Gaël James](https://github.com/gaeljames) for maintaining the package:
```

Comment:
my github profile is at https://github.com/gaelj

Reply:
Ops, fixed.